core/templates/components/button-directives/social-buttons.component.css

```diff
@@ -49,7 +49,7 @@
   padding: 5px 10px;
 }
 .oppia-footer-social .oppia-footer-social-icons .oppia-twitter-follow {
-  background: #1da1f2;
+  background: #000;
```

Comment:
Any specific reason for these changes?

Reply:
Actually this change is related to an another pr #19056